plugin/clone/include/clone_hton.h

```diff
@@ -77,6 +77,10 @@ struct Locator {
 
       m_hton = ha_resolve_by_legacy_type(thd, db_type);
 
+      DBUG_EXECUTE_IF(
+          "client_has_more_se",
+          if (m_hton->db_type == DB_TYPE_ROCKSDB) m_hton = nullptr;);
```

Comment:
We might get a little more coverage by converting db_type to DB_TYPE_UNKNOWN and moving this before the ha_resolve_by_legacy_type() call.